lib/db/db.go

```diff
@@ -0,0 +1,19 @@
+package db
+
+type Handler interface {
+	Initialize() error
+	Insert(measurements MeasurementGroup) error
+	CreateQuery(measurements MeasurementGroup) string
+	Close()
```

Comment:
looks like all implementations of this can cause errors but only print them. might want to specify that close can return an error here and actually pass through those errors to users of Handler

Reply:
Fixed